src/common/entity/cover_icon.ts

```diff
@@ -86,9 +86,9 @@ export const coverIcon = (state?: string, stateObj?: HassEntity): string => {
         case "closing":
           return mdiArrowDownBox;
         case "closed":
-          return mdiBlinds;
+          return mdiRollerShadeClosed;
```

Comment:
You made this change for both `blind` and `shade` device classes now, was that intentional?

Reply:
I'd suggest we use `blinds-horizontal` and `roller-shade`

Reply:
No it was not. I didn't realise they were one and the same.
Will have to split the case for blind device class out and leave it as original.

I assume I need to close this pull request and try again with the edits?

Reply:
No, you can continue changing this branch: <https://github.com/KablammoNick/frontend/blob/patch-1/src/common/entity/cover_icon.ts>

Reply:
Done, also noticed I missed them at the top import.

Thanks for the help.